src/containers/Profile/Profile.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { Component } from 'react';
+import { Grid, Row, Col, Nav, NavItem, Tab } from 'react-bootstrap';
+
+import { Header, Footer } from 'components';
+
+import bg from 'img/examples/city.jpg';
+import christian from 'img/christian.jpg';
+import chris1 from 'img/examples/chris1.jpg';
+import chris0 from 'img/examples/chris0.jpg';
+import chris3 from 'img/examples/chris3.jpg';
+import chris4 from 'img/examples/chris4.jpg';
+import chris5 from 'img/examples/chris5.jpg';
+import chris7 from 'img/examples/chris7.jpg';
+import chris9 from 'img/examples/chris9.jpg';
+import chris6 from 'img/examples/chris6.jpg';
+import chris8 from 'img/examples/chris8.jpg';
+// import bg from 'img/city2.jpg';
+// import bg from 'img/city2.jpg';
+
+class Profile extends Component {
+  state = { activePill: 'studio' };
+
+  handlePill = key => {
+    this.setState(() => {
+      return {
+        activePill: key
+      };
+    });
+  };
+
+  render() {
+    const { activePill } = this.state;
+    return (
+      <div className="profile-page">
+        <Header />
+
+        <div className="wrapper">
+          <div
+            className="header header-filter"
+            style={{
+              backgroundImage: `url(${bg})`,
+              backgroundSize: 'cover',
+              backgroundPosition: 'top center'
+            }}
+          />
+
+          <div className="main main-raised">
+            <div className="profile-content">
+              <Grid>
+                <Row>
+                  <div className="profile">
```

Comment:
Great!

You will need to refactor and create more components of this type, I will try to mount as much as I can